requirements.txt

```diff
@@ -3,4 +3,5 @@
 jinja2==2.6
 numpy==1.6.1
 Paver==1.2.4
-pep8==1.7.0
\ No newline at end of file
+pep8==1.7.0
+httplib2==0.10.3
```

Comment:
Can we use one of the existing http request libraries?

Reply:
What are we using at the moment?

Reply:
https://github.com/the-blue-alliance/the-blue-alliance/blob/master/datafeeds/datafeed_fms_api.py#L186

The `context` thing is a bit funny, but it allows us to make async calls.